sos/report/plugins/systemd.py

```diff
@@ -24,36 +24,36 @@ class Systemd(Plugin, IndependentPlugin):
     def setup(self):
 
         self.add_file_tags({
-            '/etc/systemd/journald.conf.*': 'insights_etc_journald_conf',
-            '/usr/lib/systemd/journald.conf.*': 'insights_usr_journald_conf_d',
-            '/etc/systemd/system.conf': 'insights_systemd_system_conf',
-            '/etc/systemd/logind.conf': 'insights_systemd_logind_conf'
+            '/etc/systemd/journald.conf': 'etc_journald_conf',
+            '/usr/lib/systemd/journald.conf': 'usr_journald_conf_d',
+            '/etc/systemd/system.conf': 'systemd_system_conf',
+            '/etc/systemd/logind.conf': 'systemd_logind_conf'
         })
 
         self.add_cmd_output([
-            "systemctl status --all",
-            "systemctl show --all",
-            "systemctl show *service --all",
-            # It is possible to do systemctl show with target, slice,
-            # device, socket, scope, and mount too but service and
-            # status --all mostly seems to cover the others.
+            "journalctl --list-boots",
+            "ls -lR /lib/systemd",
+            "systemctl list-dependencies",
+            "systemctl list-jobs",
+            "systemctl list-machines",
             "systemctl list-units",
-            "systemctl list-units --failed",
             "systemctl list-units --all",
-            "systemctl list-unit-files",
```

Comment:
We *stop* collecting `"systemctl list-unit-files"`, is that intentional?

Reply:
No it's below, I originally brought all the commands out into separate add_cmd_output calls. When I found out just adding cmd_as_tag works perfectly. I added them all back in alphabetical order, it's an ocd thing I do sometimes. I can undo all the alphabetical ordering if you think it's not needed.

Reply:
Ah, I see. So then there is just a typo :)

-            "systemctl list-unit-files",
+            "systemctl list-units-files",

Whereas `systemctl list-unit-files` does work but `systemctl list-units-files` doesnt.

Ordering does not matter at all :)

Reply:
Good catch, I corrected it.